graphql/inputs/transaction-inputs.graphql

```diff
@@ -10,6 +10,14 @@ input CreateContributionPlanTransactionInput {
     transaction_details: TransactionDetailsInput!
 }
 
+"""
+ Input used in creating a Wallet Transaction
+"""
+input CreateWalletTransactionInput {
```

Comment:
This input should not exist. Use the CreateTransactionInput, pass in the wallet_id as the owner_id, and the transaction details as usual

Reply:
The create Transaction input is for all transactions, Wallet_id can't be passed into it because there's nothing as such in the database. It breaks everything and all other transactions fail except we have to restructure the whole codebase. This is the reason why I created a separate Wallet Transaction input like others.